src/api/db/seeds.rb

```diff
@@ -47,6 +47,7 @@
 downloader_role = Role.where(title: 'downloader').first_or_create
 reader_role     = Role.where(title: 'reader').first_or_create
 
+# users table seeds won't be used in rspec
```

Comment:
It you are writing test and look the seeds you may expect the users to be loaded, so it may help to avoid confusion.